aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -0,0 +1,439 @@
+                .roleArn(role.getRoleArn())
+                .featureName(role.getFeatureName())
+                .build();
+    }
+
+    public static ListTagsForResourceRequest listTagsForResourceRequest(final String arn) {
+        return ListTagsForResourceRequest.builder()
+                .resourceName(arn)
+                .build();
+    }
+
+    public static AddTagsToResourceRequest addTagsToResourceRequest(final String arn, final Collection<Tag> tags) {
+        return AddTagsToResourceRequest.builder()
+                .resourceName(arn)
+                .tags(translateTagsToSdk(tags))
+                .build();
+    }
+
+    public static RemoveTagsFromResourceRequest removeTagsFromResourceRequest(final String arn, final Collection<Tag> tags) {
+        return RemoveTagsFromResourceRequest.builder()
+                .resourceName(arn)
+                .tagKeys(tags.stream().map(Tag::getKey).collect(Collectors.toSet()))
+                .build();
+    }
+
+    public static CloudwatchLogsExportConfiguration buildTranslateCloudwatchLogsExportConfiguration(Collection<String> previousLogExports, Collection<String> desiredLogExports) {
+        final Set<String> logTypesToEnable = new HashSet<>(Optional.ofNullable(desiredLogExports).orElse(Collections.emptyList()));
+        final Set<String> logTypesToDisable = new HashSet<>(Optional.ofNullable(previousLogExports).orElse(Collections.emptyList()));
+
+        logTypesToEnable.removeAll(Optional.ofNullable(previousLogExports).orElse(Collections.emptyList()));
+        logTypesToDisable.removeAll(Optional.ofNullable(desiredLogExports).orElse(Collections.emptyList()));
+
+        return CloudwatchLogsExportConfiguration.builder()
+                .enableLogTypes(logTypesToEnable)
+                .disableLogTypes(logTypesToDisable)
+                .build();
+    }
+
+    public static DeleteDbInstanceRequest deleteDbInstanceRequest(final ResourceModel model, final String finalDBSnapshotIdentifier) {
+        final DeleteDbInstanceRequest.Builder builder = DeleteDbInstanceRequest.builder()
+                .dbInstanceIdentifier(model.getDBInstanceIdentifier());
+        if (StringUtils.isEmpty(finalDBSnapshotIdentifier)) {
+            builder.skipFinalSnapshot(true);
+        } else {
+            builder.skipFinalSnapshot(false)
+                    .finalDBSnapshotIdentifier(finalDBSnapshotIdentifier);
+        }
+        return builder.build();
+    }
+
+    public static ResourceModel translateDbInstanceFromSdk(final software.amazon.awssdk.services.rds.model.DBInstance dbInstance, final List<software.amazon.awssdk.services.rds.model.Tag> tags) {
+
+        final String dbParameterGroupName = Optional.ofNullable(dbInstance.dbParameterGroups())
+                .orElse(Collections.emptyList())
+                .stream()
+                .map(Translator::translateDBParameterGroupFromSdk)
+                .findFirst().orElse(null);
+
+        // https://rds-jira.amazon.com/browse/WS-293
```

Comment:
Should we remove this comment as this tool is for internal use only? And add explanation why we are doing this?